management/server/core/local-peer/local-peer-impl/src/main/java/io/subutai/core/localpeer/impl/LocalPeerImpl.java

```diff
@@ -2266,6 +2271,11 @@ public UsedNetworkResources getUsedNetworkResources() throws PeerException
 
         for ( final ResourceHost resourceHost : resourceHostSet )
         {
+            if( !resourceHost.isConnected() )
```

Comment:
Can not approve this check because environment workflow needs each RH to participate even if it is offline, the results are checked later and relevant action is taken.  If at least one RH is offline, workflow must not proceed because when it gets online later, this can result in duplicated resources reserved while it was offline.